tabular/src/autogluon/tabular/models/fasttext/hyperparameters/parameters.py

```diff
@@ -0,0 +1,25 @@
+__all__ = ["get_param_baseline"]
+
+
+# https://fasttext.cc/docs/en/python-module.html#train_supervised-parameters
+def get_param_baseline():
+    params = {
+        "lr": 0.1,  # learning rate [0.1]
+        "dim": 100,  # size of word vectors [100]
+        "ws": 5,  # size of the context window [5]
+        "epoch": 50,  # number of epochs [5]
```

Comment:
is there any early stopping / warm start functionality? This could be used to control for time_limit and possibly get improved results.

Reply:
See previous comment